tests/unit/conftest.py

```diff
@@ -25,6 +24,28 @@
     from starlite import Starlite
     from starlite.types import HTTPResponseBodyEvent, HTTPResponseStartEvent, HTTPScope
 
+# In case we run unit tests without extra dependencies installed
+try:
+    from saq.job import Job
+
+    from starlite_saqlalchemy import worker  # pylint: disable=ungrouped-imports
+
+    @pytest.fixture()
+    def job() -> Job:
+        """SAQ Job instance."""
+        return Job(function="whatever", kwargs={"a": "b"})
+
+    @pytest.fixture(scope="session", autouse=True)
+    def _patch_worker() -> abc.Iterator:
+        monkeypatch = pytest.MonkeyPatch()
+        monkeypatch.setattr(worker.Worker, "on_app_startup", MagicMock())
+        monkeypatch.setattr(worker.Worker, "stop", MagicMock())
+        yield
+        monkeypatch.undo()
+
+except ModuleNotFoundError:
+    pass
```

Comment:
I think we can handle this without the try/except:

```suggestion
@pytest.fixture()
def job() -> Job:
    """SAQ Job instance."""
    if settings.IS_SAQ_INSTALLED is False:
        pytest.skip("SAQ not available")
    from saq.job import Job
    return Job(function="whatever", kwargs={"a": "b"})

@pytest.fixture(scope="session", autouse=settings.IS_SAQ_INSTALLED)
def _patch_worker() -> abc.Iterator:
    from starlite_saqlalchemy import worker
    monkeypatch = pytest.MonkeyPatch()
    monkeypatch.setattr(worker.Worker, "on_app_startup", MagicMock())
    monkeypatch.setattr(worker.Worker, "stop", MagicMock())
    yield
    monkeypatch.undo()
```
By calling `pytest.skip()` it should mean that any tests that depend on that fixture will not run.

Reply:
Wasn't aware of `pytest.skip`, it looks like the proper way of doing that!